apps/src/code-studio/pd/teachercon1819_registration/TravelPlans.jsx

```diff
@@ -126,7 +132,9 @@ export default class TravelPlans extends Teachercon1819FormComponent {
         </FormGroup>
 
         <FormGroup>
-          {this.radioButtonsFor("howTraveling")}
+          {this.radioButtonsWithAdditionalTextFieldsForFor("howTraveling", {
+            'I will carpool with another TeacherCon attendee (Please note who)': 'carpooling_with_attendee'
```

Comment:
this should probably be implemented as an exported content via TextFields, and `foodAllergy` can probably be removed from the same